README.md

```diff
@@ -1,70 +1,116 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# 📗 Table of Contents
 
```

Comment:
**[REQUIRED]** As requested by the previous reviewer, please add the `Acknowledgments` section on your README and give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

For example:
![image](https://user-images.githubusercontent.com/38088818/217912942-c6b259ec-b548-489a-8c7e-6cea45692ace.png)
 
**[OPTIONAL]** Here are key points to consider when documenting your README.
Note: This is only optional because It wasn't mentioned by the previous reviewer, but I recommend that you follow these points every time you document your README.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✔️
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✔️  
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" or "coming soon" than delete it. ✖️ 
***You are missing the `Future Features` and `Acknowledgments` sections.***
- Future Features section should include 1-3 potential features to be added. ✖️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✔️ 
For example:
![image](https://user-images.githubusercontent.com/38088818/216551345-4cb3cf19-9d8d-47d5-95be-ac8d47de0049.png)

- The License suggestion should be removed. ✔️ 
- There should not be any blank sections that do not have content. ✔️  
- There should not be any broken links in the README file. ✔️ 
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✔️